torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1074,6 +1074,26 @@ def sample_inputs_fliplr_flipud(op_info, device, dtype, requires_grad):
     )
     return [SampleInput(tensor) for tensor in tensors]
 
+def sample_inputs_logit(op_info, device, dtype, requires_grad):
+    low, high = op_info.domain
+
+    # Note: Operator is very sensitive at points near the
+    # start and end of domain and leads to NaN for float16
+    # if domain_eps is 1e-5.
+    domain_eps = op_info._domain_eps if dtype != torch.float16 else 3e-2
+
+    low = low + domain_eps
+    high = high - domain_eps
+
+    samples = (
+        SampleInput(make_tensor((S, S, S), device, dtype, low=low, high=high, requires_grad=requires_grad)),
+        SampleInput(make_tensor((S, S, S), device, dtype, low=low, high=high, requires_grad=requires_grad), args=(0.2,)),
+        SampleInput(torch.rand((), device=device, dtype=dtype, requires_grad=requires_grad)),
```

Comment:
Why rand and not make tensor here? Follow-up question: how is this row different from from the last row?

Reply:
That was a duplicate thanks.
Have consistently used make_tensor. Thanks!

**Note**: 

The [issue](https://github.com/pytorch/pytorch/issues/51242) of args and kwargs not being passed in test_unary_ufuncs test suite also takes place in case of `logit`. Since `eps` is not a mandatory argument, there is no error.